frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/import-providers-tab/providers/vm-import-password.tsx

```diff
@@ -105,11 +108,9 @@ const stateToProps = (state, { wizardReduxID, provider }) => {
     (!validationField || !isValidationCritical);
 
   if (provider === VMImportProvider.OVIRT && hasAllPrerequisiteValuesFiled) {
-    hasAllPrerequisiteValuesFiled = !!iGetOvirtFieldValue(
-      state,
-      wizardReduxID,
-      OvirtProviderField.CERTIFICATE,
-    );
+    hasAllPrerequisiteValuesFiled =
+      !!iGetOvirtFieldValue(state, wizardReduxID, OvirtProviderField.CERTIFICATE) &&
+      !iGetOvirtFieldAttribute(state, wizardReduxID, OvirtProviderField.CERTIFICATE, 'validation');
```

Comment:
this should do the same check like it is doen for API URL field with isValidationCritical. So it is easier to refactor the validator with different validation types in the future

Reply:
done